include/boolector_factory.h

```diff
@@ -14,6 +14,8 @@
 **
 **/
 
+#pragma once
```

Comment:
I made the PR https://github.com/makaimann/smt-switch/pull/83 to use it in every header file